src/components/Apps/AccountListTable/const.js

```diff
@@ -18,6 +19,7 @@ export const connectivityMeta = {
       ok: 'text-primary',
       err: 'text-danger',
       auth_err: 'text-danger',
+      rdp_err: 'text-danger',
       password_err: 'text-danger',
       openssh_key_err: 'text-danger',
       ntlm_err: 'text-danger',
```

Comment:
The provided responses appear to be examples of markdown formatting intended for use in code snippets rather than text queries. The codes you've included are not related to the specific task at hand.

If you're referring specifically to programming coding style, there may have been some miscommunication or misunderstanding due to the format used here, but if it's about best practices in code formatting (including comments and spacing), I'd say they follow conventional JavaScript conventions:

```
export const connectivityMeta = {
    // Other similar lines...
};
```